src/cephadm/cephadm

```diff
@@ -5403,7 +5410,10 @@ def _get_parser():
         help='ceph.keyring to pass through to the container')
     parser_shell.add_argument(
         '--mount', '-m',
-        help='mount a file or directory under /mnt in the container')
+        help=("mount a file or directory in the container. "
+              "Support multiple mounts passed as a comma separated list "
+              "ie: `--mount /foo,/bar:/bar`. "
```

Comment:
what do you thing of using [nargs="*"](https://docs.python.org/3/library/argparse.html#nargs) ?

Reply:
@sebastian-philipp I guess `nargs='+'` is what we need, right?

Reply:
don't know actually.